common/lib/protocol-definitions/src/consensus.ts

```diff
@@ -77,8 +77,6 @@ export interface IQuorumProposals extends IEventProvider<IQuorumProposalsEvents>
     has(key: string): boolean;
 
     get(key: string): any;
-
-    getApprovalData(key: string): ICommittedProposal | undefined;
```

Comment:
`ICommittedProposal` is another good candidate for elimination if we implement appropriate detached container behavior for the Quorum (e.g. auto-approve when detached, similar to consensus DDSs)